io.github.bit_team.back_in_time.gui.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Docu:
+        https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html
+
+    Validate (lint) on shell:
+        appstreamcli validate *xml
+-->
+
+
+<component type="desktop-application">
+  <id>io.github.bit_team.back_in_time.gui</id>
```

Comment:
There are strict and hard rules about the creation of such an `<id>`. It is derived from an existing website.
We could also use `io.readthedocs.*` or `org.codeberg.translate.*`. But the current solution was recommended by the AppStream online generator I used. I just added the `.gui` at the end to keep the opportunity open to devide between "common", "qt" and other components in the future.